app.js

```diff
@@ -138,37 +138,56 @@ sitemap(config, (sitemap, urls) => {
 				if (!!error.pageNotFound && !!record) {
 					pages.deleteObject(record.objectID, (error, result) => {
 						console.log('%d - Deleted %s:%s (%s)', id, record.objectID, record.lang, record.url);
+						tearDown();
 					});
 				}
-				tearDown();
 				return;
 			}
 			
-			pages.saveObject(record, (error, result) => {
-				if (!!error) {
-					console.log();
-					if (!!result && !!result.message) {
-						console.error('%d - Error! %s', id, result.message);
-						errors.push(result);
+			if (record.action === 'delete') {
+				pages.deleteObject(record.objectID, (error, result) => {
+					if (!!error) {
+						console.log();
+						if (!!result && !!result.message) {
+							console.error('%d - Error! %s', id, result.message);
+							errors.push(result);
+						}
+						if (!!error && !!error.message) {
+							console.error('%d - Error! %s', id, error.message);
+							errors.push(error);
+						}
+					} else {
+						console.log('%d - Action Deleted %s:%s (%s)', id, record.objectID, record.lang, record.url);
```

Comment:
Remove "Action". The line were we save is simply "Saved"

Reply:
Done